src/app/cases/styles.tsx

```diff
@@ -14,25 +8,35 @@ export const FiltersContainer = styled.div`
   gap: 1rem;
 `;
 
-export const MainDisplay = styled.main`
+export const PageContainer = styled.div`
+  height: 100vh;
+  width: 100vw;
   display: grid;
-  grid-template-columns: 5fr 10fr;
-  margin-top: 2rem;
-  width: 100%;
+  grid-template-columns: 1fr;
+  grid-template-rows: 0.2fr 1fr;
+  padding: 1rem 2rem 0rem 2rem;
+  overflow: hidden;
 `;
 
-// cards
 export const CardColumn = styled.div`
-  display: flex;
+  overflow-y: scroll;
```

Comment:
can you put `overflow-y: auto` instead? again, windows issue, but if it's `scroll`, the scrollbar will be permanently there, whereas `auto` only adds it if necessary